infrastructure/toolchain/stateless-stack.ts

```diff
@@ -21,5 +25,25 @@ export class StatelessStack extends cdk.Stack {
       pipelineName: 'OrcaBus-StatelessWorkflowManager',
       cdkSynthCmd: ['pnpm install --frozen-lockfile --ignore-scripts', 'pnpm cdk synth'],
     });
+
+    // notification for success/failure
+    const alertsBuildSlackConfigArn = ssm.StringParameter.valueForStringParameter(
+      this,
+      '/chatbot_arn/slack/alerts-build'
```

Comment:
Shouldn't that be `/chatbot/slack/umccr/alerts-arn`?

Reply:
Um, I'm using the existing param here, Flo. I think it makes sense in its own way... as it is chatbot ARN for alerts-build channel.

Reply:
Ah, OK. 
I was assuming we use the same naming across all accounts, but now realise that bastion uses a different notation. 
All good then! Keep that one.